test/functional/core/job_spec.lua

```diff
@@ -312,6 +312,24 @@ describe('jobs', function()
     end)
   end)
 
+  it('does not repeat output with slow output handlers', function()
+    source([[
+      let d = {'data': []}
+      function! d.on_stdout(job, data, event) dict
+        call add(self.data, a:data)
+        sleep 200m
+      endfunction
+      if has('win32') || has('win64')
```

Comment:
FWIW, `has('win32')` is guaranteed to always return true for all supported Windows versions.